Makefile

```diff
@@ -203,7 +203,8 @@ VK_OBJ= Kernel/Clause.o\
         Kernel/Matcher.o\
         Kernel/MaximalLiteralSelector.o\
         Kernel/SpassLiteralSelector.o\
-        Kernel/ELiteralSelector.o\
+  Kernel/ELiteralSelector.o\
```

Comment:
Indentation is off. GNU make might be OK with this, others would be in their rights to whinge. If I recall POSIX correctly...ignore if I misremembered, you are the main make-user!